src/test/groovy/org/gradle/android/PluginApplicationTest.groovy

```diff
@@ -35,10 +35,12 @@ class PluginApplicationTest extends AbstractTest {
             .writeProject()
 
         expect:
-        def result = withGradleVersion(Versions.latestSupportedGradleVersionFor(notLatestPatchAndroidVersion).version)
+        def result = withGradleVersion(TestVersions.latestSupportedGradleVersionFor(notLatestPatchAndroidVersion).version)
             .withProjectDir(projectDir)
             .withArguments("tasks", "--stacktrace")
             .build()
-        result.output =~ /WARNING: Android plugin ${quote("3.6.1")} has not been tested with this version of the Android cache fix plugin, although it may work.  We test against only the latest patch release versions of Android Gradle plugin: ${Versions.SUPPORTED_ANDROID_VERSIONS.join(", ")}.  If 3.6.1 is newly released, we may not have had a chance to release a version tested against it yet.  Proceed with caution.  You can suppress this warning with with -Dorg.gradle.android.cache-fix.ignoreVersionCheck=true./
+        result.output.readLines().findAll {
+            it =~ /WARNING: Android plugin ${quote("3.6.1")} has not been tested with this version of the Android cache fix plugin, although it may work.  We test against only the latest patch release versions of Android Gradle plugin: ${Versions.SUPPORTED_ANDROID_VERSIONS.join(", ")}.  If 3.6.1 is newly released, we may not have had a chance to release a version tested against it yet.  Proceed with caution.  You can suppress this warning with with -Dorg.gradle.android.cache-fix.ignoreVersionCheck=true./
+        }.size() == 1
```

Comment:
was this previously printing the output more than once?

Reply:
Yep, once for the app module, once for the library module.